strongbox-web-core/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+server.port = 48080
```

Comment:
Is this a Spring Boot-specific properties file?

Reply:
yes

Reply:
BTW, with SpringBoot we can easily store our configuration properties in git now